workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/CodeGeneration/Utils/Formatting.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Improbable.Gdk.CodeGeneration.Utils
+{
+    public static class Formatting
```

Comment:
Can you refactor these to take `Identifiers`? (I mean to do this in the dotnetcore version)

Reply:
I think so - will need to check we don't call them anywhere without an `Identifier`

Reply:
Discussed offline - this will be dependent on the fate of the `Identifier`. Will leave for now.